docker/packager/binary/Dockerfile

```diff
@@ -91,6 +91,9 @@ ENV PATH="$PATH:/usr/local/go/bin"
 ENV GOPATH=/workdir/go
 ENV GOCACHE=/workdir/
 
+RUN curl https://raw.githubusercontent.com/matus-chochlik/ctcache/1194532/clang-tidy-cache -Lo /usr/bin/clang-tidy-cache \
```

Comment:
`1194532` is it a commit sha prefix?
Looks unsafe.

Replace to the whole 160-bit commit sha.

Reply:
How do we check that we are on x86_64 here?